subsys/bluetooth/host/classic/a2dp.c

```diff
@@ -76,35 +173,713 @@ struct bt_a2dp *get_new_connection(struct bt_conn *conn)
+	if (err) {
+		LOG_DBG("AVDTP discover failed");
+	}
+
+	return err;
+}
+
+int bt_a2dp_configure_endpoint(struct bt_a2dp *a2dp, struct bt_a2dp_endpoint *endpoint,
+		struct bt_a2dp_endpoint *peer_endpoint,
+		struct bt_a2dp_endpoint_config *config)
+{
+	/* todo: see the API description in a2dp.h */
+	return 0;
+}
+
+int bt_a2dp_deconfigure(struct bt_a2dp_endpoint *endpoint)
+{
+	/* todo: see the API description in a2dp.h */
+	return 0;
+}
+
+static int bt_a2dp_start_cb(struct bt_avdtp_req *req)
+{
+	struct bt_a2dp *a2dp = START_PARAM(START_REQ(req));
+	struct bt_a2dp_endpoint *endpoint;
+	struct bt_a2dp_endpoint_internal *ep_internal;
+
+	endpoint = CONTAINER_OF(a2dp->start_param.lsep, struct bt_a2dp_endpoint, info);
+	ep_internal = bt_a2dp_get_endpoint_state(endpoint);
+	if (ep_internal == NULL) {
+		return -EPERM;
+	}
+
+	LOG_DBG("START result:%d", a2dp->start_param.status);
+	if (!a2dp->start_param.status) {
+		if (endpoint->control_cbs.start_play != NULL) {
+			endpoint->control_cbs.start_play(0);
+		}
+	} else {
+		if (endpoint->control_cbs.start_play != NULL) {
+			endpoint->control_cbs.start_play(a2dp->start_param.status);
+		}
+	}
+	return 0;
+}
+
+int bt_a2dp_start(struct bt_a2dp_endpoint *endpoint)
+{
+	struct bt_a2dp *a2dp;
+	struct bt_a2dp_endpoint_internal *ep_internal;
+
+	if (endpoint == NULL) {
+		return -EPERM;
+	}
+
+	ep_internal = bt_a2dp_get_endpoint_state(endpoint);
+	if (ep_internal == NULL) {
+		return -EPERM;
+	}
+
+	a2dp = ep_internal->a2dp;
+	if (a2dp == NULL) {
+		return -EPERM;
+	}
+
+	a2dp->start_param.req.func = bt_a2dp_start_cb;
+	a2dp->start_param.acp_stream_endpoint_id = ep_internal->remote_ep_info.id;
+	a2dp->start_param.lsep = &endpoint->info;
+	if (bt_avdtp_start(&a2dp->session, &a2dp->start_param) != 0) {
+		if (endpoint->control_cbs.start_play != NULL) {
+			endpoint->control_cbs.start_play(-1);
+		}
+		return -EIO;
+	}
+	return 0;
+}
+
+int bt_a2dp_stop(struct bt_a2dp_endpoint *endpoint)
+{
+	/* todo: see the API description in a2dp.h */
+	return 0;
+}
+
+int bt_a2dp_reconfigure(struct bt_a2dp_endpoint *endpoint,
+		struct bt_a2dp_endpoint_config *config)
+{
+	/* todo: see the API description in a2dp.h */
+	return 0;
+}
+
+uint32_t bt_a2dp_get_media_mtu(struct bt_a2dp_endpoint *endpoint)
+{
+	struct bt_a2dp_endpoint_internal *ep_internal;
+
+	if (endpoint == NULL) {
+		return -EPERM;
+	}
+
+	ep_internal = bt_a2dp_get_endpoint_state(endpoint);
+	return bt_avdtp_get_media_mtu(&ep_internal->endpoint->info);
+}
+
+#if defined(CONFIG_BT_A2DP_SOURCE)
+struct net_buf *bt_a2dp_media_buf_alloc(struct net_buf_pool *pool)
+{
+	struct net_buf *buf;
+	uint8_t *send_buf_header;
+
+	buf = bt_l2cap_create_pdu(pool, 0);
+	if (buf == NULL) {
+		LOG_DBG("bt_l2cap_create_pdu fail");
+		return buf;
+	}
+
+	send_buf_header = net_buf_add(buf, sizeof(struct bt_avdtp_media_hdr));
+	memset(send_buf_header, 0, sizeof(struct bt_avdtp_media_hdr));
+	((struct bt_avdtp_media_hdr *)(send_buf_header))->playload_type = 0x60U;
```

Comment:
You can consider using a macro to represent this value and passing the macro to this variable, instead of using the number directly. It's easier to modify it.